internal/base64_json/base64_json.go

```diff
@@ -29,6 +29,12 @@ func keySort(keys []string) {
 	})
 }
 
+// Encode returns a pointer to the base64 representation of the result of
+// marshalling v. If an error occurs in the process, it is returned.
+// Under the hood, Encode transforms v into a *orderedmap.OrderedMap so
+// the result will always wrap a key-sorted JSON representation to this
+// package. If v's underlying type is already *orderedmap.OrderedMap, v's
+// wrapped value will be used as is.
```

Comment:
```suggestion
// Encode returns a pointer to the base64 representation of the result of
// marshalling v in JSON. If an error occurs in the process, it is returned.
// Under the hood, Encode transforms v into a *orderedmap.OrderedMap so
// the result will always wrap a key-sorted JSON representation.
// If v's underlying type is already *orderedmap.OrderedMap, v's
// wrapped value will be used as is.
```

Reply:
`to this package` is here to remind that the key order is not a lexicographical one, but a custom one, internal to the package `base64_json`

Thoughts?

Reply:
Maybe `the result will always wrap JSON representation in which keys are sorted in a specific order defined by this package.` or something like that. I did not understand the previous wording 😅

Reply:
I went for your suggestion. Figured as this package is internal, there's no need for such precision. Everything should have been updated.